cmd/gbounty/bootstrap/update.go

```diff
@@ -22,11 +23,40 @@ import (
 	"github.com/bountysecurity/gbounty/kit/slices"
 )
 
+func getLastCheckTime() (time.Time, error) {
+	filePath, err := lastCheckFilePath()
+	if err != nil {
+		return time.Time{}, err
+	}
+
+	data, err := os.ReadFile(filePath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return time.Time{}, nil
+		}
+		return time.Time{}, err
+	}
+
+	return time.Parse(time.RFC3339, string(data))
+}
+func checkForUpdatesRequired() bool {
```

Comment:
I think there's enough noise already, so I'd prefer to remove the calls to `pterm.Info.Println` within this function.